client/source/components/questionsAnswers/LoadAddQA.jsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 import { LoadContainer } from './styles/Containers.styled';
 
-const LoadAddQA = function CreateLoadAddQAComponent() {
+const LoadAddQA = function CreateLoadAddQAComponent({
+  questionCount, setQuestionCount, getQuestions
+}) {
   return (
     <LoadContainer>
-      <button type="button" data-testid="More answered" onClick={() => {}}>More Answered Questions</button>
+      <button
```

Comment:
This "More Answers" button needs a conditional. Per the BRD, it lists "Once all of the questions for the product have been loaded, the button should no longer appear."

Reply:
Yes, on the todo list!